src/components/editors/layoutEditor.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable object-curly-newline */
+import React from 'react'
+
+import { Typography, Grid } from '@material-ui/core'
+import { Dropdown, Slider } from '../SettingComponent'
+import { OPTIONS } from '../../lib/options'
+
+import './layoutEditor.css'
+
+const LayoutEditor = () => (
+  <Grid
+    container
+    direction="column"
+    justify="flex-start"
+    alignItems="flex-start"
+    spacing={5}
+  >
+    <Grid item>
+      <Typography align="center">Overlay</Typography>
+      <div className="someOption">
+        <Typography className="someOptionName">Justification</Typography>
```

Comment:
This usage of passing through the options, you can instead iterate over the options you want in this file and render the component type that is defined in the options for it, and pass through all the props (that's what the `DynamicOptions` component is all about in Desktop. You might even be able to replace all of these different editors with 1 `DynamicOption` component, and then just define the lists of options you want per tab.